build.sh

```diff
@@ -188,7 +188,7 @@ if [ "$PIPELINE" = 'build' ]; then
         time docker pull "$GAME_IMAGE" || true
     fi
     date
-    time docker build \
+    STEAM_USERNAME="$STEAM_USERNAME" STEAM_PASSWORD="$STEAM_PASSWORD" time docker build \
```

Comment:
reverted `export DOCKER_BUILDKIT=1` to original position

Reply:
sorry, i forced push, thought i was making the change.

Reply:
better for  `DOCKER_BUILDKIT=1` to be on this line, for better specificity that it pertains only to `docker build` than a global `export`.

Reply:
`hlds` working.

Reply:
better to use the global for more consistency in `docker` userd

Reply:
`DOCKER_BUILDKIT=1` is only relevant to `docker build` and `docker buildx ...` commands

Reply:
e.g. https://github.com/startersclan/docker-hlstatsxce-daemon/actions/runs/4397801757/jobs/7701187270#step:10:121

Reply:
that's for logs where verbosity may be preferred. can't say the same should be used in scripts

Reply:
even though this is a long imperative script, the way it is writen with `\` for each line is a declarative programming approach, so long flags will be more readable.